pkg/generate/app/cmd/resolve.go

```diff
@@ -108,7 +106,10 @@ func Resolve(r *Resolvers, c *ComponentInputs, g *GenerationInputs) (*ResolvedCo
 	for _, repo := range repositories {
 		repo.SetContextDir(g.ContextDir)
 	}
-
+	// Environment File where the parameters are stored
+	if len(g.EnvironmentFile) != 0 && !app.IsPossibleEnvironmentFile(g.EnvironmentFile) {
+		return nil, fmt.Errorf("when --env-file is specified you must provide a valid env file location")
+	}
```

Comment:
@bparees, I believe we need this because if the user gives a file which doesn't exist(unintentionally or intentionally), we should not create a new pod. Without this validation, a new pod will be created.